.flake8

```diff
@@ -53,6 +53,7 @@ per-file-ignores =
     gui/wxpython/gui_core/ghelp.py: E722
     gui/wxpython/gui_core/gselect.py: F841, E266, E722
     gui/wxpython/gui_core/preferences.py: E266, F841
+    gui/wxpython/gui_core/pyedit.py: E501
```

Comment:
These issues need to be resolved in the file at the specific line where they occur. Adding this line here would ignore all problems in a the given file. We are trying to reduce number of items in this list in the other PRs. Something like `# noqa: E501` ([In-line Ignoring Errors](https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html#in-line-ignoring-errors)).

Reply:
Ok, I see.

Reply:
New attempt in 1f572e5e898a461c398a104e1fc073604bd41fc3